databricks/koalas/groupby.py

```diff
@@ -1010,6 +1010,101 @@ def rank(self, method='average', ascending=True):
         """
         return self._rank(method, ascending)
 
+    # TODO: add keep= parameter
+    def nsmallest(self, n=5):
```

Comment:
Can you move this under the class `SeriesGroupBy`? Seems this is `Series`-only API.

Reply:
It's under `SeriesGroupBy.nsmallest`. I hava remove it from `DataFrameGrouBy`.

Reply:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.SeriesGroupBy.nsmallest.html?highlight=nsmallest#pandas.core.groupby.SeriesGroupBy.nsmallest